### PR TITLE
Fix #2228 SCEditor - Duplicate tags after re-election.

### LINKED ISSUE #2228: SCEditor - Duplicate tags after re-election.
Hi, 

For some time now users reporting to me bug in editor.  For example: if first select (ex. font-size), then again change this font - it not change BBCode, but adding next.

GIF presents: 

It's hard to do with so many modify tags on tapatalk for example...

Original thread: [SCEditor - Duplicate tags after re-election.](http://community.mybb.com/thread-184162.html)

### PR DESCRIPTION
**Pull request by martec**
fix #2228

### CODE REVIEW

File: jscripts/bbcodes_sceditor.js [hunk 1]
```diff
@@ -78,46 +78,55 @@ $(document).ready(function($) {
...
 			}
-
-			return '[size=' + size + ']' + content + '[/size]';
-		},
-		html: function(token, attrs, content) {
-			return '<span data-scefontsize="' + attrs.defaultattr + '" style="font-size:' + attrs.defaultattr + '">' + content + '</span>';
+			if (size <= 0) {
+				size = 1;
+			}
+			return '<font data-scefontsize="' + attrs.defaultattr + '" size="' + size + '">' + content + '</font>';
```
Comment (Destroy666x) [inline jscripts/bbcodes_sceditor.js]: Why is it changed to outdated `<font>`?

Comment (martec) [inline jscripts/bbcodes_sceditor.js]: you need ask to developer of sceditor. Core of plugin use font, so if use span here, issue will happen. 
https://github.com/samclarke/SCEditor/blob/master/src/plugins/bbcode.js#L2283

File: install/resources/mybb_theme.xml [hunk 2]
```diff
@@ -13738,9 +13738,9 @@ if(use_xmlhttprequest == "1")
 <td class="trow1" width="20%"><strong>{$lang->username}</strong></td>
 <td class="trow1">{$mybb->user['username']} <span class="smalltext">[<strong><a href="member.php?action=logout&amp;logoutkey={$mybb->user['logoutkey']}">{$lang->change_user}</a></strong>]</span></td>
 </tr>]]></template>
-		<template name="codebuttons" version="1806"><![CDATA[<link rel="stylesheet" href="{$mybb->asset_url}/jscripts/sceditor/editor_themes/{$theme['editortheme']}" type="text/css" media="all" />
+		<template name="codebuttons" version="1807"><![CDATA[<link rel="stylesheet" href="{$mybb->asset_url}/jscripts/sceditor/editor_themes/{$theme['editortheme']}" type="text/css" media="all" />
 <script type="text/javascript" src="{$mybb->asset_url}/jscripts/sceditor/jquery.sceditor.bbcode.min.js?ver=1805"></script>
-<script type="text/javascript" src="{$mybb->asset_url}/jscripts/bbcodes_sceditor.js?ver=1804"></script>
+<script type="text/javascript" src="{$mybb->asset_url}/jscripts/bbcodes_sceditor.js?ver=1807"></script>
```
Comment (Destroy666x) [inline install/resources/mybb_theme.xml]: Should also be bumped in ACP's Users module

File: jscripts/bbcodes_sceditor.js (hunk 1 above)
Comment (Destroy666x) [inline jscripts/bbcodes_sceditor.js]: Ok, I see

File: install/resources/mybb_theme.xml (hunk 2 above)
Comment (martec) [inline install/resources/mybb_theme.xml]: thanks...
done.

File: jscripts/bbcodes_sceditor.js (hunk 1 above)
Comment (Destroy666x) [inline jscripts/bbcodes_sceditor.js]: But that makes `xx-small` and `x-small` the same size, doesn't it? Indexes are 0 and 1, so the conditional will make them both 1. This needs to be fixed.

I guess I'll report this to SCEditor team sot hat they change it to better `<span>`.

**Comment by martec** [comment]
PR updated. @Destroy666x please check this.

**Comment by Cu8eR** [comment]
@Destroy666x is possible to merge this?

**Comment by dequeues** [comment]
`xx-small` and `x-small` still seem to be the same size.

**Comment by martec** [comment]
??

[img]https://i.imgur.com/FqTFeNf.png[/img]
[img]https://i.imgur.com/yNgLcZz.png[/img]

**Comment by dequeues** [comment]
Guess it was something on my end. Checked again and it's fine. Apologies.

**Pull request merged**